ProcessMaker/processmaker
# FOUR-13980 Fix Case title overlaps task name in saved searches

## Issue & Reproduction Steps
The content of the case title column is stepping on the task name column in a save search

![image](https://github.com/ProcessMaker/processmaker/assets/8028650/6dec6c25-5c02-496a-abe9-028b40a829bf)

## Solution
- Fix the text trucation and enable always the text truncation for case title and task name.

![image](https://github.com/ProcessMaker/processmaker/assets/8028650/854617d9-b5e1-4b08-985f-7db57931f57c)

## How to Test
- Configure the Case Title of a process (Put a large text).
- Run the process
- Go to the All requests default Saved Search, or create a new one
- See the content of the case_title column

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-13980

## Code Review Checklist

ci:next
ci:deploy
